content/guides/embed/ui-elements/theming-branding.md

```diff
@@ -0,0 +1,92 @@
+---
+rank: 15
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/explorer
+  - embed/ui-elements/picker
+  - embed/ui-elements/uploader
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+
+# Themes & Branding for Box UI Elements
+
+With branding & themes for Box UI Elements you can customize the look and
+behavior of your embedded Box component to your enterprise's requirements.
+The tool provides a real-time live preview for an easy adjustment of the theme.
+
+## How to start
+
+Add the selected BUIE to your application and choose one of the default Box
+themes templates.
+
+## Customization
+
+You can change these elements in your embedded Box component.
```

Comment:
Please add intro about design tokens. If possible it'd be nice to include sample snippets.